src/Adapter/OAuth2.php

```diff
@@ -323,6 +326,29 @@ public function authenticate()
         return null;
     }
 
+    /**
+     * {@inheritdoc}
+     *
+     * Checking access_token only works for oauth1 and oauth2, openid will overwrite this method.
```

Comment:
I think we don't need this description - since it will inheritdoc. Pls fix above